### ISSUE TITLE
restructure the approach to read in the Noah MP parameter table

### ISSUE THREAD

**Issue by HelinWei-NOAA** [NOAA-EMC/global-workflow#1614]
**Description**
Noah MP parameter table is part of the source code in the current version. The UFS model has to be recompiled whenever there is any parameter change in the table. Therefore we propose to change this structure and the table will become one of the input file to the model.

**Requirements**
The Noah MP table has to be put somewhere in the global workflow and the running script needs to specify the path of this file (softlink or copy this file to the running directory) 

**Acceptance Criteria (Definition of Done)**
I suggest this table file should be put into the directory parm. But I don't know which subdirectory? Or create a new one para_land?

ush/forecast_postdet.sh needs to be modified to link or copy this table file

**Comment by aerorahul** [comment]
@HelinWei-NOAA Thank you for opening this issue.
Is this file/table part of the ufs-weather-model repository?  If so, we can pick directly from there into the global-workflow.

**Comment by HelinWei-NOAA** [comment]
@aerorahul Yes. It can be part of the ufs-weather-model repository (under ccpp-physics)
> @HelinWei-NOAA Thank you for opening this issue. Is this file/table part of the ufs-weather-model repository? If so, we can pick directly from there into the global-workflow.

**Comment by HelinWei-NOAA** [comment]
@aerorahul What part of global workflow do we need to modify to accomplish this goal? Now the table is sitting in the ccpp-physics repo.

**Comment by WalterKolczynski-NOAA** [comment]
@HelinWei-NOAA An entry needs to be added to `sorc/link_workflow.sh` to copy/link the appropriate file(s) from UFS to somewhere in `parm` (we can discuss where). Then one of the scripts for the forecast job (likely `ush/forecast_postdet.sh` or `ush/parsing_namelists_FV3.sh`) will need to be updated to copy the file from `parm` to the working directory when the forecast job runs. If workflow needs to allow multiple options for this file, a setting can be added to the `fcst` and `efcs` config files.

Are these tables templates that need to have information substituted at runtime, or are they static?

**Comment by HelinWei-NOAA** [comment]
@WalterKolczynski-NOAA Can we create a sub_dir para_land under parm and put this file here? The table is static and there is no information substituted at runtime.

**Comment by WalterKolczynski-NOAA** [comment]
Create a new `parm/ufs/land` directory and put the new file(s) there.

**Comment by HelinWei-NOAA** [comment]
@WalterKolczynski-NOAA I have created [a branch](https://github.com/HelinWei-NOAA/global-workflow/tree/hr2_land1_table) for this PR. How can I link it here? Or I need to create a new PR?

**Comment by WalterKolczynski-NOAA** [comment]
Depends.

- If you just want to link it, you can just drop a link.

- If you mention the issue in your commit message, that activity will appear here:
```
Do something

What the thing is and why thing was done.

Refs #1614
```

- When you open a PR, you should attach it to the issue by using closing keywords in the description:
```
Description of what the PR does and why.

Resolves #1614
```
